sdcm/sct_config.py

```diff
@@ -1215,6 +1215,22 @@ class SCTConfiguration(dict):
         'region_name', 'n_db_nodes', 'ami_id_db_scylla', 'ami_id_loader'
     ]
 
+    stress_cmd_params = [
+        # this list is used for variouse checks against stress commands, such as:
+        # 1. Check if all c-s profile files existing that are referred in the commands
+        # 2. Check what stress tools test is needed when loader is prepared
+        'stress_cmd', 'stress_read_cmd', 'stress_cmd_w', 'stress_cmd_r', 'stress_cmd_m',
+        'prepare_write_cmd', 'stress_cmd_no_mv', 'stress_cmd_no_mv_profile',
+        'prepare_stress_cmd', 'stress_cmd_1', 'stress_cmd_complex_prepare', 'prepare_write_stress',
+        'stress_cmd_read_10m', 'stress_cmd_read_cl_one', 'stress_cmd_read_80m',
```

Comment:
I think gemini also should be here: `gemini_cmd`

Reply:
fixed